Directory.Build.props

```diff
@@ -22,7 +22,6 @@
     <BaseIntermediateOutputPath>$(BaseArtifactsPath)obj</BaseIntermediateOutputPath>
     <DebugType>embedded</DebugType>
     <EnableDefaultNoneItems>false</EnableDefaultNoneItems>
-    <Nullable>enable</Nullable>
```

Comment:
I've just seen this in the commit history - we aren't using NRTs for C#?

Reply:
Not right now. `Ink` isn't using them itself and I don't want to block this PR on figuring out the correct manual bindings.

Reply:
Ah okay, understandable.